vg/font.go

```diff
@@ -220,6 +225,31 @@ func getFont(name string) (*truetype.Font, error) {
 	return font, err
 }
 
+func absFontPath(fontpath string) (string, error) {
+	ext := filepath.Ext(fontpath)
+	if ext == "" {
+		fontpath += ".ttf"
+	} else if ext != ".ttf" {
+		return "", errors.New("Only support font type ttf")
+	}
+
+	if filepath.IsAbs(fontpath) {
+		if _, err := os.Lstat(fontpath); err != nil {
+			return "", err
+		}
+		return fontpath, nil
+	} else {
```

Comment:
"else" clause is not necessary since you return, so change this to '}' and reduce the indenting.